contracts/test/poolBooster/metropolis-pool-booster.sonic.fork-test.js

```diff
@@ -21,7 +22,7 @@ describe("ForkTest: Metropolis Pool Booster", function () {
     oSonicVault = fixture.oSonicVault;
     oSonic = fixture.oSonic;
     poolBoosterFactoryMetropolis = fixture.poolBoosterFactoryMetropolis;
-    governor = fixture.governor;
+    governor = await impersonateAndFund(addresses.multichainStrategist);
```

Comment:
Same comment as above about renaming the variable

Reply:
[424a4a4](https://github.com/OriginProtocol/origin-dollar/pull/2611/commits/424a4a45665a525eacad838a54cb7f5b75c8f8ba)